`make image` is failing

```
Dockerfile:34
--------------------
  32 |     # We need .git and app/ in order to get the version and git version for the frontend/.env file
  33 |     # that's generated when building the frontend.
  34 | >>> COPY ./.git /headlamp/.git
  35 |     COPY app/package.json /headlamp/app/package.json
  36 |
--------------------
ERROR: failed to solve: source can't be a git ref for COPY
make: *** [Makefile:99: image] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -31,7 +31,8 @@ FROM --platform=${BUILDPLATFORM} node:18 as frontend-build
 
 # We need .git and app/ in order to get the version and git version for the frontend/.env file
 # that's generated when building the frontend.
-COPY ./.git /headlamp/.git
+COPY .git/ ./headlamp/.git/
+
 COPY app/package.json /headlamp/app/package.json
 
 # Keep npm install separated so source changes don't trigger install

